katsdpcontroller/generator.py

```diff
@@ -1122,7 +1131,7 @@ def _make_vis_writer(g, config, name, s3_name, local, prefix=None, max_channels=
     vis_writer.cpus = min(3, 2 * info.n_vis / _N32_32)
 
     workers = 4
-    max_accum_dumps = _writer_max_accum_dumps(config, name)
+    max_accum_dumps = _writer_max_accum_dumps(config, name, 10, max_channels)
```

Comment:
Perhaps add a comment (or use a constant) to indicate the meaning of `bytes_per_vis` here.

Reply:
Done.